hail/python/hail/methods/misc.py

```diff
@@ -91,7 +91,7 @@ def maximal_independent_set(i, j, keep=True, tie_breaker=None, keyed=True) -> Ta
 
     When multiple nodes have the same degree, this algorithm will order the
     nodes according to ``tie_breaker`` and remove the *largest* node.
-    
```

Comment:
whitespace change, and not a fix to something against default formatting?

Reply:
My editor does this automatically. I can revert this in particular if we want, but also I feel that it's good style to not have trailing whitespace.

Reply:
yeah, I'm sympathetic, but it does make the diff harder too read. A setting that applies the transformation only in lines otherwise modified would be great -- does that exist?